wren-ui/src/apollo/server/adaptors/ibisAdaptor.ts

```diff
@@ -32,40 +32,25 @@ export type IbisPostgresConnectionInfo =
   | UrlBasedConnectionInfo
   | HostBasedConnectionInfo;
 
-export interface IbisMySQLConnectionInfo {
-  host: string;
-  port: number;
-  database: string;
-  user: string;
-  password: string;
-}
-
-export interface IbisSqlServerConnectionInfo {
-  host: string;
-  port: number;
-  database: string;
-  user: string;
-  password: string;
-}
-
 export interface IbisBigQueryConnectionInfo {
   project_id: string;
   dataset_id: string;
   credentials: string; // base64 encoded
 }
 
 export type IbisConnectionInfo =
+  | UrlBasedConnectionInfo
+  | HostBasedConnectionInfo
```

Comment:
The connection info of MySQL, Postgres, MS SQL, and ClickHouse are similar and can be classified into host-based or URL-based connection info